scancode-analyzer.about

```diff
@@ -0,0 +1,7 @@
+about_resource: .
```

Comment:
I think we always use .ABOUT extension uppercase as a convention

Reply:
Yes! Changing this.